components/monitoring/README.md

```diff
@@ -41,159 +41,117 @@ for adding the service monitor declaration:
                    `openshift.io/cluster-monitoring: 'true'`. Otherwise, it will not be
                    monitored by the user workload Prometheus instance.
 
-#### 3. Grafana dashboards
+### 3. Grafana dashboards
 
 A dashboard is a set of one or more panels organized and arranged into one or more rows. It visualizes results from multiple data sources simultaneously.
 New dashboards can be added through the user interface, preconfigured in infra-deployment project, or imported from other projects.
 
-##### Manual export
+#### Create a dashboard
 
-  - Create a new folder in [grafana](https://grafana-appstudio-workload-monitoring.apps.appstudio-stage.x99m.p1.openshiftapps.com) for your service. (In the left nav, + Create Folder)
-
-  - [Create a dashboard](https://grafana.com/docs/grafana/v9.0/dashboards/) for your team's view of your service's Service Level Indicators. (After navigating to your folder, + Create Dashboard)
+  - [Create a dashboard](https://grafana.com/docs/grafana/v9.0/dashboards/) for your team's view of your service's Service Level Indicators. (After navigating to your folder, + Create Dashboard)  
+  ***Note:***  
+  Creating a new dashboard is available only in development environment.
+  You may consider editing the existing `example` dashboard For staging and production environments  
+  
 
-  - Add tiles on the dashboard to track your initial set of service level indicators. If you correctly added your servicemonitor to the stage Prometheus datasource, it will show up in the Query list when you edit a tile.
+  - Add tiles to the dashboard to track your initial set of service level indicators. If you correctly added your servicemonitor to the stage Prometheus datasource, it will show up in the Query list when you edit a tile.
```

Comment:
```suggestion
  - Add tiles to the dashboard to track your initial set of service level indicators. If you correctly added your
   servicemonitor to the stage Prometheus datasource, it will show up in the Query list when you edit a tile.
```